GoogleForCreators/web-stories-wp
# Design System Storybook: More Icons

## Summary
A few icons were added to [figma ](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3441%3A80829)after I initially imported the svgs. Just getting current.

## Relevant Technical Choices



## To-do



## User-facing changes

None, only in storybook

## Testing Instructions

Verify the following icons are added to the designSystem part of storybook

In Action: 
- grid (updated not new)
- gridMany
- gridSingle
- save

In Editor:
- placeholder
- reverseColor
- rotateColor
- staggerFromLeft
- staggerFromRight

In Navigation: 
- doubleChevron


---



Fixes #
